lua/entities/gmod_wire_damage_detector.lua

```diff
@@ -104,7 +104,9 @@ end
 
 function ENT:LinkEnt( ent, dontupdateoutput )
 	if self.linked_entities_lookup[ent] then return false end
-
+	
+	if not ent or not ent:IsValid() then return end
```

Comment:
Could you make this the first line of the function?

Reply:
Sure thing